workflow-examples/jobs-in-parallel/jobs_in_parallel.groovy

```diff
@@ -0,0 +1,10 @@
+def branches = [:]
+for (int i = 0; i < 4; i++) {
+  def which = "${i}" 
```

Comment:
it is
I had to do it
the reason is that jenkins will not run the job multiple times if it gets the same params so I had to use a "dummy" parameter just to start the job with different params each time.